Maman11/Q2/my_suffix.c

```diff
@@ -0,0 +1,45 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#include <string.h>
+#define STRING_LENGTH 127
+
+int suffix(char str[], char c){
+    bool read = false;
+    int suffix = 0, len = strlen(str), i = 0;
+    while (str[i]){
```

Comment:
I want you to iterate over the string in a cleaner way.
In c you can do amazing stuff using pointers.
Try to create. achar* pointer to the string and with this pointer iterate over the string.
DO this using a. char* pointer and a for loop

ALso this about a way to know when you are at the end of a sting. (`\0`)


I will help you more.
str[] is an array of chars in the memory. Each char is a byte. 
So str will look like this in the memory - 0000 (this is the start address): byte, byte. byte .......


So you can create a pointer for the first byte and then iterate over the bytes